arduino-ide-extension/src/common/protocol/sketches-service-client-impl.ts

```diff
@@ -178,7 +180,17 @@ export class SketchesServiceClientImpl
     if (toCheck.scheme === 'user-storage') {
       return false;
     }
-    if (READ_ONLY_FILES.includes(toCheck?.path?.base)) {
+
+    const isCloudSketch = toCheck
+      .toString()
+      .includes(path.join(REMOTE_SKETCHBOOK_FOLDER, ARDUINO_CLOUD_FOLDER));
```

Comment:
This will always evaluate as `false` on Windows because `toCheck` uses POSIX path separator (`/`). For example:

```text
file:///c%3A/Users/per/AppData/Local/Arduino15/RemoteSketchbook/ArduinoCloud/e1552df2-7538-4927-a99c-bdba1d315467/Untitled_jan02a/thingProperties.h
```

but `path.join` uses the native path separator, which is the annoying `\` on Windows:

```text
RemoteSketchbook\ArduinoCloud
```

Reply:
ah, that makes sense. Fixing...